chip/mchp/lfw/ec_lfw.c

```diff
@@ -410,6 +422,16 @@ void lfw_main(void)
 	uart_init();
 	system_init();
 
+	/*
+	 * We need to switch control of VCI_OUT (aliased as EC_ON) away from
+	 * VCI_INx to keep the machine powered even after the user releases the
+	 * power button. This ensures that we can stay on long enough to read
+	 * from SPI flash.
+	 */
+	gpio_reset(GPIO_EC_ON);
```

Comment:
This might seem counterintuitive, but `gpio_reset` _is_ documented as turning off alternative states for the GPIO and asserting the flags set in `gpio.inc`; this results in it being re-muxed as GPIO and pulled high.